content/en/docs/openshift-portability/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Differences between Microshift and OKD"
+weight: 100
+main_menu: true
+content_type: concept
+description: "Addressing some innate differences between OKD and Microshift."
+---
+# Differences Between OKD and Microshift
+
+## App Portability
+
+Microshift is, by design, a trimmed down version of OKD 4.x and as such, comprimises are made to affect the resource footprint of the platform's runtime, streamlining of the platform's distribution, and to tune Microshift to edge use-cases.  By and large, this should not impeded application portability between OKD 4.x and Microshift, but it is worth noting key differences.
+
+### Where's all the Operators?
```

Comment:
This sounds quite colloquial. I believe we should stick to a more concise, factual voice.